src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/advice/ErrorMessageTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers.advice;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+class ErrorMessageTest {
+
+    @Test
+    void should_set_properties() {
+
+        ErrorMessage testErrorMessage = new ErrorMessage(
+            new Exception("some message"),
+            HttpStatus.INTERNAL_SERVER_ERROR,
+            LocalDateTime.now()
+        );
+
+        assertEquals("Internal Server Error", testErrorMessage.getError());
+        assertEquals("some message", testErrorMessage.getMessage());
+        assertEquals(500, testErrorMessage.getStatus());
```

Comment:
I wonder if we need the http status in the error body if we have it in the overall response already.